ionic2/app/components/poke-poi-card/poke-poi-card.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="slide-card" #slideCard @slide="slideState">
+  <div class="poke-poi-card-content" *ngIf="pokePOI" [ngClass]="'poi-type-'+pokePOI.getType()">
+    <ion-row>
+
+      <ion-col class="poke-column">
+        <ion-row>
+          <poke-poi-bubble [pokePOI]="pokePOI"></poke-poi-bubble>
+        </ion-row>
+        <ion-row>
+          <poke-type [type]="pokePOI.pokemon.types"></poke-type>
+        </ion-row>
+      </ion-col>
+
+      <ion-col style="flex-grow: 1;">
+        <ion-row>
+          <span class="poke-name">{{ pokePOI.pokemon.name }}</span>
+          <span class="poke-id">#{{ pokePOI.pokemon.pokemonId }}</span>
+        </ion-row>
+        <ion-row class="poke-description">
+          {{ pokePOI.pokemon.description }}
+        </ion-row>
+
+
+        <ion-row class="action-button-bar">
+          <button clear (click)="launchDirections()">
+            <ion-icon name="navigate"></ion-icon>
+            Directions
+          </button>
+
+          <button clear (click)="launchPokeDex()">
+            <ion-icon name="book"></ion-icon>
+            PokeDex
+          </button>
+        </ion-row>
+      </ion-col>
+    </ion-row>
+
+    <div class="poke-poi-type-line"></div>
```

Comment:
![poicard old](https://cloud.githubusercontent.com/assets/2697771/18988882/203ea614-8709-11e6-9181-3e1fce62e890.png)


As you can see in the review, I removed the type-line and the rounded corner.